samcli/cli/hidden_imports.py

```diff
@@ -39,4 +39,5 @@ def walk_modules(module: HasPathAndName, visited: set) -> None:
     "aws_lambda_builders.workflows",
     "configparser",
     "dateparser",
+    "jsonschema",
```

Comment:
What will happen if we don't add this line?

Reply:
https://ci.appveyor.com/project/AWSSAMCLI/aws-sam-cli-n3bhc/builds/45853277